src/ploomber/cli/install_ploomber.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+which pythona > /dev/null 2>&1
+PYTHON_EXIT_STATUS=$?
+if [ "$PYTHON_EXIT_STATUS" -ne "0" ] 
+then
+    echo 'Python not found. Installing Python...'
+
+    which condaa > /dev/null 2>&1
```

Comment:
typo?

Reply:
Yeah that's a typo. I was testing the which command. I'll tag you when it's done since the CI/CD pipeline is still failing.